### PR TITLE
Add plpython and postgresql-docs classes

### PR DESCRIPTION
**Pull request by DavidS**
This is a straight-forward rebase/squash of https://github.com/puppetlabs/puppetlabs-postgresql/pull/569

### CODE REVIEW

File: manifests/globals.pp [hunk 1]
```diff
@@ -1,55 +1,57 @@
...
 
-  $manage_pg_hba_conf   = undef,
-  $manage_pg_ident_conf = undef,
-  $manage_recovery_conf = undef,
+  $manage_pg_hba_conf     = undef,
+  $manage_pg_ident_conf   = undef,
+  $manage_recovery_conf   = undef,
 
-  $manage_package_repo  = undef
+  $manage_package_repo    = undef,
```
Comment (cyberious) [inline manifests/globals.pp]: additional spaces will cause lint issue

**Comment by cyberious** [comment]
I think this might have been more aggressive squash than what we needed, it should have kept all of the previous commit messages within the body of the one commit with a summary on the first line

**Comment by DavidS** [comment]
The other commit messages only referred to changes to the original commit. The only thing I might have dropped inadvertently is that "and tests" part.

File: manifests/globals.pp (hunk 1 above)
Comment (DavidS) [inline manifests/globals.pp]: `rake lint` passes this locally.

**Comment by cyberious** [comment]
I was talking more about the body of the commit message not the summary line. Similar to https://github.com/puppetlabs/puppetlabs-registry/commit/868e77e4613b7eed2e48b464e19bd546cdced56c were we have a summary and more details as to why the change

**Comment by DavidS** [comment]
Updated the commit description.

**Pull request merged**